kola/tests/misc/omaha.go

```diff
@@ -76,7 +75,6 @@ func OmahaPing(c cluster.TestCluster) error {
 
 	select {
 	case <-tc:
-		platform.Manhole(m)
```

Comment:
Manhole is just a special case debugging mechanism, if you aren't working on that specific test it isn't going to be clear why kola opened a shell and is waiting for input.